Control/Monad/Logger.hs

```diff
@@ -49,6 +49,7 @@ module Control.Monad.Logger
     , filterLogger
     , NoLoggingT (..)
     , mapNoLoggingT
+    , LogLine(..)
```

Comment:
This is a type alias, it should be exported without the `(..)`

Reply:
Indeed, and I merely wrote it like this because I did so for `LogSource` as well, under the assumption that it "couldn't hurt".